do-not-commit/disabled-cert-validation.py

```diff
@@ -0,0 +1,16 @@
+import requests
+import requests as req
+
+some_url = "https://example.com"
+
+# ok
+r = req.get(some_url, stream=True)
+# ok
+r = requests.post(some_url, stream=True)
+
+# ruleid:disabled-cert-validation
+r = req.get(some_url, stream=True, verify=False)
```

Comment:
Certificate verification has been explicitly disabled. This
permits insecure connections to insecure servers. Re-enable
certification validation. ([y.disabled-cert-validation.disabled-cert-validation](https://semgrep.dev/r/y.disabled-cert-validation.disabled-cert-validation))
#
<div>
<a href="https://semgrep.dev">
<img align="right" src="https://r2c.dev/semgrep_icon_text.svg" height=28px alt="" />
</a>

<sub> [How do I suppress this?](https://semgrep.dev/docs/running-rules/#managing-findings) **·** [Report a bug](https://github.com/returntocorp/semgrep-rules/issues/new?assignees=&labels=&template=rule_bug_report.md&title=y.disabled-cert-validation.disabled-cert-validation) **·** [Manage rule](https://semgrep.dev/manage/projects)</sub>
</div>